### PR TITLE
Fix SQL keywords breaking the queries

### LINKED ISSUE #34: SQL syntax in customvar breaks the cube
## Expected Behavior
Does not break

## Current Behavior
image

image

### PR DESCRIPTION
**Pull request by theFeu**
fixes #34

### CODE REVIEW

File: library/Cube/DbCube.php [hunk 1]
```diff
@@ -130,11 +130,10 @@ public function getDbName()
      *
      * @return \Zend_Db_Select
      */
-    public function innerQuery()
+    public function getInnerQuery()
```
Comment (Thomas-Gelf) [inline library/Cube/DbCube.php]: Please be careful when changing public methods in the Cube. The Cube provides Hooks for custom Actions (see `library/Cube/Hook/ActionsHook.php' ). Those Hooks receive the Cube instance, and public methods on the Cube make part of the API they might be expecting. Blind guess: this change breaks the Cube Action Hook implementation of the Icinga Director

Comment (theFeu) [reply library/Cube/DbCube.php]: Oh, I wasn't aware of that.
Thanks for pointing it out!

I will revert the name refactor :)

File: library/Cube/Ido/IdoHostStatusCube.php [hunk 2]
```diff
@@ -91,6 +91,7 @@ public function prepareInnerQuery()
 
         $view->applyFilter($this->getMonitoringRestriction());
 
+        $select = $view->getQuery()->getSelectQuery();
```
Comment (lippserd) [inline library/Cube/Ido/IdoHostStatusCube.php]: We don't need this twice, do we? 😆

Comment (theFeu) [reply library/Cube/Ido/IdoHostStatusCube.php]: Oof, how did that happen?
Sorry, I'll pay better attention to the diffs next time!

File: library/Cube/DbCube.php [hunk 3]
```diff
@@ -165,7 +164,12 @@ public function finalizeInnerQuery()
             }
         }
 
-        $query->columns($columns + $this->factColumns);
+        $c = [];
+        foreach ($columns + $this->factColumns as $k => $v) {
+            $c[$this->connection->getDbAdapter()->quoteIdentifier($k)] = $v;
```
Comment (lippserd) [inline library/Cube/DbCube.php]: Either use `$this->connection->getDbAdapter()` or just `$this->db()` but not both.

**Comment by lippserd** [comment]
Please squash your commits.

**Pull request merged**